src/Core/Process/Factor.fs

```diff
@@ -32,7 +32,7 @@ type Factor =
 
     /// Get ISATab string entries from an ISAJson Factor object
     static member toString (factor : Factor) =
```

Comment:
Should this also be renamed to "toStringOption"

Reply:
I think i named it "toStringObject", but yes we should rename. Not sure to what.